src/elements/core/styles/mixins/lists.scss

```diff
@@ -76,28 +76,119 @@
   }
```

Comment:
Please cleanup structure to avoid warnings on yarn build (https://github.com/sbb-design-systems/lyne-components/actions/runs/10630456007/job/29469418012?pr=3038)